File: src/practice/luogu/p5711/Main.go
// https://www.luogu.com.cn/problem/P5711
package main

import "fmt"

func main() {
	var i int
	fmt.Scanf("%d", &i)
	if (i%4 == 0 && i%100 != 0) || i%400 == 0 {
		fmt.Println(1)
	} else {
		fmt.Println(0)
	}
}

File: src/practice/luogu/p5711/Main.cpp
// https://www.luogu.com.cn/problem/P5711

#include <cstdio>

int main() {
  int i;
  scanf("%i", &i);
  printf("%i\n", (i % 4 == 0 && i % 100 != 0) || i % 400 == 0 ? 1 : 0);
  return 0;
}